include/ibm/management_console_rest.hpp

```diff
@@ -1135,6 +1174,15 @@ inline void requestRoutes(App& app)
                const std::shared_ptr<bmcweb::AsyncResp>& asyncResp) {
                 handleBroadcastService(req, asyncResp);
             });
+
+    BMCWEB_ROUTE(app, "/ibm/v1/OCC/Control/<str>/Actions/PassThrough.Send")
+        .privileges({{"ConfigureComponents", "ConfigureManager"}})
```

Comment:
Can we go admin only? 
Drop ConfigureComponents ? 
Sorry not in the requirements, just noticed now

Reply:
Done